ruby/packagescanner.go

```diff
@@ -137,7 +137,7 @@ func (ps *Scanner) Scan(ctx context.Context, layer *claircore.Layer) ([]*clairco
 			Name:           name,
 			Version:        version,
 			Kind:           claircore.BINARY,
-			PackageDB:      g,
+			PackageDB:      `ruby:` + g,
```

Comment:
This change probably won't show up for existing manifests with a scanner version update, is it needed for matching? Do you think it warrants a version update?